### ISSUE TITLE
change to specialties being produced under ideal conditions

### ISSUE THREAD

**Issue by NicolaRavani** [MagicBees/MagicBees#129]
followup on #128 
I found now documentation, other than that issue report, for the change to the production. Neither could I find a possibility to change it in the config files.

This change is in my opinion highly counterintuitiv in combination with gendustry.

I got a setup with almost all bees and cant see their preferences in the beealyzer because I changed them with gendustry. This means I have to lookup all bees either in wikis or rebreed them.

If it is easy to realize, please offer a config setting to ignore conditions for special products and note those changes in your changelogs. Thank you very much for your effort!

**Comment by mezz** [comment]
Sorry for the lack of communication about this issue, it was something that was fixed during some major code changes, and I was not aware that it was broken earlier.

I think you may still have some misconceptions about this. Gendustry's Industrial Apiaries or Forestry's Alvearies have no problem supporting bees with special products. The only temperature that can't be simulated in the Alveary is the "hellish" temperature, but Gendustry has no problem with that.

**Comment by NicolaRavani** [comment]
You are right about the simple possibility with gendustry or even alvearies to support those needs.
I am kinda okay with the change, i mean i love the mod either way. The fact that temperature and humidity is kinda pointless to manipulate just bugs me a little.
I guess i was just a little frustrated at first, because it took me a while to find out where the problem even was.

Thanks for the respond and the your good work in general!
